src/common/validation.tsx

```diff
@@ -38,7 +38,7 @@ export const getRandomNumbers = (min: number, max: number) => {
 };
 
 export const validateName = (name: string) => {
-  const pattern = /^([a-zA-Z]*( [a-zA-Z])?)+$/;
+  const pattern = /^[A-Za-z\u00C0-\u017F'-][A-Za-z\u00C0-\u017F\s'-]*$/;
```

Comment:
Can do the former, but let's not do the latter as it would just be partial internationalization (we would need to support other non-latin scripts as well if we go that route, if we ever do).